src/components/alert/alertCenter.svelte

```diff
@@ -120,13 +121,15 @@
         {alert.content}
         <div slot="actions">
           {#each alert.actions as action}
-            <Button
+            <svelte:component
+              this={action.component || Button}
               size={alert.mode === "full" ? "medium" : "small"}
               fab={action.icon && !action.text}
               kind={action.kind || 'filled'}
               isDisabled={action.isDisabled}
               isLoading={action.isLoading}
               onClick={() => action.action(alert)}
+              {...action}
```

Comment:
Possibly, this should go first and we could remove the `isDisabled` and `isLoading` props?

Reply:
Sure. For my own learning, though, why?

Reply:
'cause it's being passed in via the spread anyway

we're doing 

```
const action = { isDisabled: false, isLoading: true }
const props = { isDisabled: action.isDisabled, isLoading: action.isLoading, otherProps, ...action }

// when we could do
const props = { ...action, otherProps }
```